cmd/mr_list.go

```diff
@@ -105,6 +115,8 @@ func init() {
 		&mrAssignee, "assignee", "", "list only MRs assigned to $username")
 	listCmd.Flags().StringVar(&order, "order", "updated_at", "display order, updated_at(default) or created_at")
 	listCmd.Flags().StringVar(&sortedBy, "sort", "desc", "sort order, desc(default) or asc")
+	listCmd.Flags().BoolVarP(&mrDraft, "draft", "", false, "list MRs marked as draft")
+	listCmd.Flags().BoolVarP(&mrReady, "ready", "", false, "list MRs not marked as draft")
 
```

Comment:
What if we don't use 2 flags here, but have only `--draft` to denote we want to also list draft MRs, while calling `lab mr list` with no flags always list only ready MRs?

Reply:
Forget it.. makes no sense at all.